src/main/java/studio/magemonkey/fabled/data/sql/tables/FabledPlayersSQL.java

```diff
@@ -0,0 +1,180 @@
+        super(Fabled.inst());
+        try (PreparedStatement create = SQLManager.connection()
+                .prepareStatement("CREATE TABLE IF NOT EXISTS " + Table + "("
+                        + "UUID varchar(36), "
+                        + "Data MEDIUMTEXT)")) {
+            create.execute();
+        } catch (SQLException e) {
+            Fabled.inst().getLogger().warning("[SQL:FusionPlayersSQL:FusionPlayersSQL] Something went wrong with the sql-connection: "
+                            + e.getMessage());
+        }
+    }
+
+    public PlayerAccounts loadPlayerAccounts(OfflinePlayer player) {
+        if (player == null || player.getName() == null) return null;
+        try(Connection connection = SQLManager.connection()) {
+            PreparedStatement statement = connection.prepareStatement("SELECT * FROM " + Table + " WHERE UUID = ?");
+            statement.setString(1, player.getUniqueId().toString());
+            ResultSet result = statement.executeQuery();
+            if(result.next()) {
+                return SQLUtils.load(player, result.getString("Data"));
+            }
+
+        } catch (SQLException e) {
+            Fabled.inst().getLogger().warning("[SQL:FusionPlayersSQL:loadPlayersAccounts] Something went wrong with the sql-connection: "
+                            + e.getMessage());
+        }
+
+        return null;
+    }
+
+    public Map<UUID, PlayerAccounts> loadAllOnlinePlayerAccounts() {
+        Map<UUID, PlayerAccounts> result = new HashMap<>();
+        for (Player player : Bukkit.getOnlinePlayers()) {
+            result.put(player.getUniqueId(), loadPlayerAccounts(player));
+        }
+        return result;
+    }
+
+    public void savePlayerAccounts(OfflinePlayer player, PlayerAccounts accounts) {
+        if (player == null || player.getName() == null) return;
+        try(Connection connection = SQLManager.connection()) {
+            PreparedStatement statement = connection.prepareStatement("UPDATE " + Table + " SET Data = ? WHERE UUID = ?");
+            DataSection data = IOManager.save(accounts);
+            if(data == null) {
+                Fabled.inst().getLogger().warning("[SQL:FusionPlayersSQL:savePlayersAccounts] Data is null for player: " + player.getName());
+                return;
+            }
+            statement.setString(1, data.toString());
+            statement.setString(2, player.getUniqueId().toString());
+            statement.execute();
+        } catch (SQLException e) {
+            Fabled.inst().getLogger().warning("[SQL:FusionPlayersSQL:savePlayersAccounts] Something went wrong with the sql-connection: "
+                            + e.getMessage());
+        }
+    }
+
+    public void saveAllPlayerAccounts() {
+        Map<UUID, PlayerAccounts> data = PlayerLoader.getAllPlayerAccounts();
+        for(Map.Entry<UUID, PlayerAccounts> entry : data.entrySet())
+            savePlayerAccounts(Bukkit.getOfflinePlayer(entry.getKey()), entry.getValue());
```

Comment:
Again, if we could do a batch update (executeBatch), I think that would be best.

Reply:
Reworked with the next commit too!